docs/faas/aws-lambda.md

```diff
@@ -66,6 +66,10 @@ contain an incomplete trace context which indicates X-Ray isn’t enabled. The e
 `Context` will be valid and sampled only if AWS X-Ray has been enabled for the Lambda function. A user can
 disable AWS X-Ray for the function if the X-Ray Span Link is not desired.
 
+Instrumentation SHOULD include an explicit option to use `Context` in `_X_AMZN_TRACE_ID` as parent insted of `Link`,
```

Comment:
Is it expected that instrumentations do anything special about X-Ray parents without sampled flag? If not, maybe a note could still be added that this may lead to all spans being unsampled unless X-Ray is enabled.

Reply:
Isn't that the case with any parent span context with an unset sampled flag?  If so, that's something that should probably be addressed more broadly by the spec.

Reply:
Yes it is, but with X-Ray is unusual in that you will get a valid unsampled span context if X-Ray is disabled.